### PR TITLE
Separate config and rollout; introduce ausroller module

### PR DESCRIPTION
**Pull request by mattthias**
* This commit separates the code for configuration (reading cli
  parameter, reading config file) and the actual code to rollout.

### CODE REVIEW

File: ausroller.py [hunk 1]
```diff
@@ -1,244 +1,19 @@
-#!/usr/bin/env python2
-# encoding: utf-8
+#!/usr/bin/python
```
Comment (jmastr) [inline ausroller.py]: In `core.py` you take:
```
#!/usr/bin/env python2
# encoding: utf-8
```
I suggest to be consistent here

**Review by jmastr — CHANGES_REQUESTED** [review]
Refactoring is looking. No need to import the whole Ausroller for the test case(s). Great! Please change the minor suggestion.

File: ausroller.py (hunk 1 above)
Comment (mattthias) [reply ausroller.py]: Good point. But the hashbang line is superfluous in core.py and main() too :-D 
Will fix that

**Comment by mattthias** [comment]
@jmastr  @hemarkus  Could you please review this PR again?

**Pull request merged**